src/concepts/what-is-zilla.md

```diff
@@ -0,0 +1,132 @@
+---
+
+# What is Zilla?
+
+Zilla is a multi-protocol edge and service proxy that helps streamline, secure, and manage event-driven architectures. It is meant to be deployed alongside event-driven applications and services to enforce authentication, validate schemas, gather metrics, and terminate TLS. Additionally, Zilla has advanced protocol meditation capabilities, particularly to and from Kafka.
+
+Zilla is stateless, cloud-native, and supports various network and application protocols, including HTTP, Kafka, SSE, MQTT, gRPC, and WebSocket (additional protocols are on the way).
+
+![Zilla Overview](/zilla-overview.png)
+
+::: info Just want to build?
+Jump to the [quickstart](../how-tos/quickstart/index.md) guide.
+:::
+
+## Why Zilla?
+
+Some of the hardest operational challenges inside distributed architectures relate to networking and observability. As a result, edge and service proxies are required to achieve resilient, transparent, and properly routed connectivity among distributed services.
+
+While solutions such as Envoy, HAProxy, NGINX, etc. can help unify a distributed data plane, they are designed for “mesh” deployments in which services are directly interconnected and communicate almost exclusively over HTTP. Inside event-driven architectures (EDAs) though, services are separated by an event/message broker, and multiple protocols are present. This decoupled and multi-protocol nature of EDAs presents a new class of networking, observability, and security challenges that the Zilla proxy is designed to address.
+
+## Zilla Features
+
+### Self-Contained, Stateless Architecture
+
+Zilla has no external dependencies, is stateless, and is highly memory efficient. When deployed as an edge proxy, it scales horizontally to support millions of concurrently connected clients.
+
+### HTTP/2, gRPC, SSE, MQTT, and Kafka Support + Protocol Mediation
+
+Inside Zilla, every protocol, whether it is TCP, TLS, HTTP, Kafka, gRPC, etc., is treated as a stream, so mediating between protocols simplifies to mapping protocol-specific metadata.
+
+### Protobuf, Avro, and JSON Schema payloads
+
+Zilla supports payload schema specifications for message validation and translation.
+
+### OpenAPI and AsyncAPI Support
+
+Zilla supports different API schema specifications for message validation and API creation.
+
+### API Registry and Schema Registry Integrations
+
+Zilla integrates with schema registries and API registries including Apicurio, Confluent Schema Registry in <ZillaPlus/>, and Karapace.
+
+### Security
+
+Zilla can terminate TLS and supports JWT-based authorization for REST, SSE, and MQTT endpoints/services.
+
+### Observability
+
+Zilla can expose a Prometheus metrics endpoint and logs events to stdout. Additionally Zilla supports OpenTelemetry for both metrics and logging and. Syslog and AWS Cloudwatch are supported in <ZillaPlus/>.
+
+## Zilla Use Cases
+
+Zilla can be used as a service proxy (sidecar) or as an AsyncAPI Kafka gateway.
```

Comment:
Should AsyncAPI be hyperlinked, since many folks will not already know what it is perhaps?